clang/tools/clang-offload-deps/ClangOffloadDeps.cpp

```diff
@@ -74,6 +74,24 @@ static void reportError(Error E) {
   logAllUnhandledErrors(std::move(E), WithColor::error(errs(), ToolPath));
 }
 
+// clang-offload-bundler is currently generating a 'standardized' target triple.
+// This is of the following form - Kind-Triple-TargetID.
+// Triple's format - Architecture-Vendor-OS-Environment
+// This routine transforms the target triple specified by user as input to
+// clang-offload-deps to this 'standardized' format.
+static std::string standardizedTarget(std::string OrigTarget) {
```

Comment:
I wonder why is it different from the version from `Driver.cpp`?

Reply:
Driver.cpp standardizes only the triple (middle part of the Target). Target here consists of three parts: 1. kind (sycl, host, openmp). 2. target triple (arch-vendor-os-env; not sure why they call it triple when they have 4 parts) and 3. Target ID (sycldevice, nvidia, etc). 

Thanks

Reply:
Thank you